src/cmds/play.c

```diff
@@ -14,10 +14,17 @@
 #include <drivers/audio/portaudio.h>
 #include <fs/file_format.h>
 #include <util/math.h>
+#include <framework/mod/options.h>
+
+#define PLAY_FROM_FLASH OPTION_GET(BOOLEAN, play_from_flash)
+#define FRAMES_PER_BUFFER OPTION_GET(NUMBER, frames_per_buffer)
 
 static void print_usage(void) {
 	printf("Usage: play [WAVAUDIOFILE]\n"
-	       "       play -s\n");
+	       "       play -s\n"
```

Comment:
I guess it's good idea to update manual page (in `Play.my`) for this command if you change the help message.